packages/pluggableWidgets/toggle-buttons-native/webpack.config.prod.js

```diff
@@ -1 +1,8 @@
-module.exports = require("../../../configs/webpack.prod.native.js");
+const merge = require("webpack-merge");
+const baseConfig = require("@mendix/pluggable-widgets-tools/configs/webpack.native.config.js");
+
+baseConfig[0].module.rules[2].use.options.plugins.push("@babel/plugin-transform-flow-strip-types");
```

Comment:
what happens if the order change in piw-tools?

Reply:
It breaks. That's the con of this solution. Before I had an implementation that overrides all rules:
```
const merge = require("webpack-merge");
const baseConfig = require("@mendix/pluggable-widgets-tools/configs/webpack.native.config.js");

const customConfig = {
    mode: "production",
    devtool: false,
    module: {
        rules: [
            {
                test: /\.tsx?$/,
                loader: "ts-loader",
                options: {
                    compilerOptions: {
                        target: "esnext"
                    }
                }
            },
            {
                test: /\.jsx?$/,
                exclude: /node_modules/,
                use: {
                    loader: "babel-loader",
                    options: {
                        cacheDirectory: true,
                        presets: ["@babel/preset-react"],
                        plugins: [
                            ["@babel/plugin-proposal-class-properties", { loose: true }],
                            ["@babel/plugin-transform-react-jsx", { pragma: "createElement" }]
                        ]
                    }
                }
            },
            {
                test: /\.jsx?$/,
                include: /node_modules/,
                use: {
                    loader: "babel-loader",
                    options: {
                        cacheDirectory: true,
                        presets: ["@babel/preset-react"],
                        plugins: [
                            ["@babel/plugin-proposal-class-properties", { loose: true }],
                            "@babel/plugin-transform-react-jsx",
                            "@babel/plugin-transform-flow-strip-types"
                        ]
                    }
                }
            }
        ]
    }
};

const customConfigurations = [merge.smartStrategy({ "module.rules": "replace" })(baseConfig[0], customConfig)];

const hasEditorConfig = baseConfig.length === 2;

if (hasEditorConfig) {
    customConfigurations.push(baseConfig[1]);
}

module.exports = customConfigurations;
```


We cannot just merge, because that will lead to duplicate rules which makes behaviour hard to predict.

Reply:
@diego-antonelli Do you have a suggestion?

Reply:
search for the rule and add it there otherwise if the order changes we loose the custom config

Reply:
Searching for the rule now in the implementation

Reply:
LGTM